### ISSUE TITLE
Error on accepting the chat request on the aggregation channel

### ISSUE THREAD

**Issue by abhishekkant** [tompaana/intermediator-bot-sample#50]
Hi 

I have taken the default implementation for human handoff from here. It was working fine. 

Then I added simple dialogs to the bot project. Now, when i attempt to use human handoff, I receive the following error: 
Method not found: ‘Void Microsoft.Bot.Schema.ConversationAccount…ctor(System.Nullable`1<Boolean>, System.String, System.String, System.String, System.String, System.String)’.

The above error occurs after the user from web chat requests for help and the request arrives at the aggregate channel (in our case the emulator). The error comes up when the "Accept" button on the aggregate channel is clicked. 

What could be the issue?

Some dependencies: 
Bot Builder Framework: 4.3.2
Bot Message Routing: 2.0.2-alpha
ASPNETCore: 2.2.0

Cheers
Abhishek Kant

**Comment by mailforsachin** [comment]
I am receiving the same error. 

> Sorry, it looks like something went wrong: Method not found: 'Void Microsoft.Bot.Schema.ConversationAccount..ctor(System.Nullable`1<Boolean>, System.String, System.String, System.String, System.String, System.String)'.

**Comment by tompaana** [comment]
This problem is caused by the mismatch of the Bot Builder version - the Nuget package of [Bot Message Router](https://github.com/tompaana/bot-message-routing) uses an older version. I will try update the package soon-ish, but in the meanwhile, here's what you can do:
- Remove the Bot Message Routing Nuget package
- Clone or copy the Bot Message Routing project (https://github.com/tompaana/bot-message-routing)
- Add the Bot Message Routing project into the solution
- Update the Bot Builder version used by the Bot Message Routing project

That will fix the method not found exception.

**Comment by mailforsachin** [comment]
@tompaana 

I am missing something. So I used Nuget package manager to remove the Routing package form  the solution (Step 1) then cloned the sample to the same directory (2) ... added the bot msg routing project by importing the CSPROJ file. and then updated the bot builder libraries


![Removed](https://user-images.githubusercontent.com/25243792/59219716-c2e16f80-8be0-11e9-85db-2746f463d357.PNG)

It is not able to find the assembly
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'ConversationReference' could not be found (are you missing a using directive or an assembly reference?)	BotMessageRouting	C:\Users\124708\Desktop\intermediator-bot-sample\bot-message-routing\BotMessageRouting\MessageRouting\DataStore\RoutingDataManager.cs	152	Active

**Comment by mailforsachin** [comment]
@tompaana Can you please check the snapshots I send and let us know what we need to do to fix this? Appreciate your help.

**Comment by mailforsachin** [comment]
Hi @tompaana Could you please assist to get this sorted?

**Comment by tompaana** [comment]
![image](https://user-images.githubusercontent.com/1629215/59352091-83be3600-8d28-11e9-824c-8bb9cc9bda47.png)

Worked for me.

**Comment by mailforsachin** [comment]
Thanks @tompaana  It is working as expected. But how do we integrate this with the Enterprise Bot template? We are having issues compiling.

**Comment by praveengov** [comment]
@tompaana Still, it is throwing the same issue when I updated bot framework to 4.5.1

**Comment by mailforsachin** [comment]
@praveengov Check the thread. I faced the same issue and it was resolved.

**Comment by praveengov** [comment]
> @praveengov Check the thread. I faced the same issue and it was resolved.

@mailforsachin Yes Sachin. Just now I re-published and tested. Now it is working fine!! Thanks :)

**Comment by rungsoso** [comment]
Bot Builder Framework: 4.5.1

1. Remove the Bot Message Routing Nuget package from the solution
2. Clone or copy the Bot Message Routing project (https://github.com/tompaana/bot-message-routing)
3. Open BotMessageRouting.sln Update Nuget packet Microsoft.Bot.Builder from 4.2.2 to 4.5.1 and Rebuild project
4. Add reference BotMessageRouting.dll to solution

That will fix for me

![1567062772271](https://user-images.githubusercontent.com/24908027/63919257-96632400-ca68-11e9-81e3-ed51525af571.jpg)
